src/wasm-lib/kcl/src/std/sketch.rs

```diff
@@ -891,9 +922,9 @@ pub enum PlaneData {
     },
 }
 
-impl From<PlaneData> for Plane {
-    fn from(value: PlaneData) -> Self {
-        let id = uuid::Uuid::new_v4();
+impl Plane {
```

Comment:
Out of scope for this PR, but I don't understand why `Plane`'s impl is in `sketch.rs` and not `plane.rs`

Reply:
Did you change this from a `from` implementation because to the `From<T>` trait requires a named type `T` that wasn't worth making? I agree just want to understand

Reply:
I needed to add an extra parameter. The `From` trait specifies what the signature is. It's like an interface in TS. So the extra parameter on the function means it no longer implements the trait.

Reply:
`From` is a conversion. Sometimes it makes sense to define it near the type you're converting from, even though it's an impl on the type you're converting to. This is especially true when the to-type is defined in another crate that you don't have access to. For us here, I can check this PlaneData type. If it's okay to make it public, then I can move it. But I was erring on the side of moving less code.

Reply:
I moved it https://github.com/KittyCAD/modeling-app/pull/4101/commits/430030b36971642b91eb221001f30c8c603bd77a